creator.rb

```diff
@@ -0,0 +1,67 @@
+require_relative 'association_creator'
+require_relative 'game'
+require_relative 'module/screen'
+
+class Creator
+  include Screen
+  def initialize(args)
+    @association = AssociationCreator.new(args)
+    @games = args[:games]
+    @books = args[:books]
+    @music_albums = args[:music_albums]
+  end
+
+  def create_game
+    print 'Is it Multiplayer? [Y/N]: '
+    multi_player = user_input == 'n'
+    print 'Last Played At (yyyy-mm-dd): '
+    last_played_at = user_input
+    game = Game.new(!multi_player, last_played_at, { publish_date: publish_date, archived: !archived })
+    @games << game
+    create_associations(game)
+    puts 'Game created successfully'
+  end
```

Comment:
- [ ] Let us throw an error while a user enters a wrong format date `2021-20-20`, you need to format the user_input mostly on published_date or elsewhere related to the date. Your format is (yyyy-mm-dd) so try to validate it.

Make sure the validation of format date is working in all features that have date format.

Hint:  ` 'publish_date' => @publish_date.strftime('%Y/%m/%d'), `

![date](https://user-images.githubusercontent.com/55635977/138103606-81b3e218-93cb-4eb4-88bb-36ae9ab646e4.png)

Reply:
Hi @Emmyn5600
Thank you for pointing out these

Reply:
Well solved